x/participation/types/params.go

```diff
@@ -3,19 +3,25 @@ package types
 import (
 	"errors"
 	"fmt"
+	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"gopkg.in/yaml.v2"
-
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v2"
 )
 
 var _ paramtypes.ParamSet = (*Params)(nil)
 
 var (
-	KeyAllocationPrice           = []byte("AllocationPrice")
-	KeyParticipationTierList     = []byte("ParticipationTierList")
-	KeyWithdrawalAllocationDelay = []byte("WithdrawalAllocationDelay")
+	// MinTimeframeSize defines the minimum allowed value for either `RegistrationPeriod` or `WithdrawalDelay`
+	MinTimeframeSize = int64(time.Hour.Seconds() * 24) // One day
+	// MaxTimeframeSize defines the maximum allowed value for either `RegistrationPeriod` or `WithdrawalDelay`
+	MaxTimeframeSize = int64(time.Hour.Seconds() * 24 * 21) // 21 days
```

Comment:
We hardcode here the Unbonding Period while it may change with governance or be different than 21 days.

In my humble opinion, we don't need to add such restriction in the code. We can just set the `registrationPeriod` and `withdrawalDelay` params accordingly in the idea that you can't unbonding tokens to participate in other fundraising.
Eventually when someone want to make a governance proposal to make a change in these params, they must justify the new values and if out of the bound of unbonding period, the community has the responsibility to approve or not these changes

Reply:
IMO the only restriction we should have is that both values are positive
We can't really predict the usage of the platform, if auctions are very successful only few hours for registration may be necessary

Reply:
This was not intended to restrict to the unbonding period but just to prevent unreasonable values. They way I have set these constants is in fact arbitrary. But yeah it makes sense to remove them entirely